README.md

```diff
@@ -1,19 +1,60 @@
 # .NET Source-build Reference Packages
 
-This repository contains source and build scripts for source-buildable reference versions of historical .NET Core packages that are referenced by the [source-build](https://github.com/dotnet/source-build) repo.  These are used only when building the source-build repo.
+This repository contains tools, source and build scripts for source-buildable reference
+versions of historical .NET Core packages that are referenced by
+[.NET source build](https://github.com/dotnet/source-build). These are used only
+when source building .NET.
 
-### Continuous builds
+This repo supports three package types
 
-Microsoft build outputs from this repo are published by Arcade to blob storage.
-They are used by source-build when prebuilts are acceptable to avoid rebuilding
-the repo. This repo doesn't change often, making a cache particularly useful.
+1. Reference - A package that contains API signatures, no implementation.  It enables developers to build targeting a specific library version without having the full implementation assembly for that version.
+1. Targeting - A bundle of reference packages that target a specific .NET TFM.
+1. Text only - Packages that contain text only assets.
 
-### Build on Linux or MacOS
+## Supported Platforms
 
+.NET source build currently only supports Linux.
+
+## Building
+
+``` bash
+./build.sh -sb
 ```
-./build.sh
+
+## Adding new Packages
+
+The [generate script](https://github.com/dotnet/source-build-reference-packages/blob/main/generate.sh) supports generating new packages. Run `generate.sh -h` for usage details.
+
+When generating a package(s), the tooling will detect and generate all dependent packages.
+
+### Reference
```

Comment:
Should maybe the 3rd point in [this](https://github.com/dotnet/source-build-reference-packages/pull/506#issuecomment-1412815417) comment be addressed here as well? i.e. ref packs added to SBRP before the changes that use them are in the product can get cleaned up by accident

Reply:
I added a cleanup section to cover this.